p2p/src/error.rs

```diff
@@ -17,6 +17,9 @@ pub enum SessionsError {
     /// Errors when updating sessions
     #[fail(display = "Is not an outbound consolidated peer")]
     NotOutboundConsolidatedPeer,
+    /// Errors when using SessionType::Feeler sessions in SessionsManager
+    #[fail(display = "Peer provided from feeler has been used in SessionsManager")]
```

Comment:
```suggestion
    #[fail(display = "SessionsManager was told to manage a Feeler session. The session will be ignored because Feeler sessions should not be managed.")]
```